src/components/Sidebar/Sidebar.tsx

```diff
@@ -112,6 +112,23 @@ export const Sidebar = ({ mentors, handleModal }: SidebarProps) => {
           handleModal={openModal}
         />
       </SidebarNav>
+      <a
+        href="https://vercel.com/?utm_source=[coding-coach]&utm_campaign=oss"
+        aria-label="Vercel"
+        target="_blank"
```

Comment:
For security purposes we should also include the `noopener` directive, like this:

```suggestion
        target="_blank"
        rel="noopener"
```